Remotion/Development/Mixins/Development.Mixins.csproj

```diff
@@ -15,6 +15,7 @@
     <ProjectReference Include="..\..\Core\Tools\Core.Tools.csproj" />
     <ProjectReference Include="..\..\Mixins\Core\Mixins.Core.csproj" />
     <ProjectReference Include="..\Core\Development.Core.csproj" />
+    <ProjectReference Include="..\Moq\Development.Moq.csproj" PrivateAssets="All"/>
```

Comment:
Drop.
```suggestion
```

This dependency should not be needed. Otherwise, we introduce a dependency on a particular mocking framework into a general purpose assembly.